idalib/src/idb.rs

```diff
@@ -379,6 +381,29 @@ impl IDB {
         }
     }
 
+    pub fn strings(&self) -> StringList {
+        StringList::new(self)
+    }
+
+    pub fn get_string(&self, index: usize) -> Option<String> {
+        let addr = self.strings().get_item_addr(index)?;
+        let len = self.strings().get_item_length(index);
+
+        let bytes = self.get_bytes(addr, len);
+
+        Some(String::from_utf8_lossy(&bytes).escape_debug().to_string())
```

Comment:
This could be `String::from_utf8_lossy(&bytes).into_owned()`

Reply:
Ok. It is my understanding that we still have to use `to_string()` after `escape_debug()`. Or am I missing something?

Reply:
I don’t think you need `escape_debug` at all here, if the intention is to just get a valid utf8 string.

Reply:
I see. It makes sense. I applied the escape because I wanted to be able to print strings in a similar way as IDA does in its GUI. But users of idalib can always decide to escape or not on their own. I'll modify it as soon as I get to a computer (I'm currently on mobile).